### PR TITLE
Fixed highlight for "switch" keyword

### PR DESCRIPTION
**Pull request by AsterAI**
Also
1. Updated readme to expand information about how to start develop the Zephir plugin
2. Changed depended lib to support by now days IDE

### CODE REVIEW

**Comment by AsterAI** [comment]
Download built plugin http://asterai.com/Zephir.zip

File: .gitignore [hunk 1]
```diff
@@ -1,5 +1,7 @@
 /out/
 /gen/
-src/com/zephir/lexer/ZephirLexer.java
 *~
 *.jar
+/Zephir.iml
+/Zephir.zip
+/.idea
```
Comment (sergeyklay) [inline .gitignore]: For these purposes create a global .gitignore file, which is a list of rules
for ignoring files in every Git repository on your computer.

https://help.github.com/articles/ignoring-files/#create-a-global-gitignore

File: .gitignore [hunk 2]
```diff
@@ -1,5 +1,7 @@
 /out/
 /gen/
-src/com/zephir/lexer/ZephirLexer.java
 *~
 *.jar
+/Zephir.iml
```
Comment (sergeyklay) [inline .gitignore]: Could you please explain a bit more - why?

**Comment by AsterAI** [comment]
@sergeyklay  Sure, have fixed

**Comment by AsterAI** [comment]
@sergeyklay  Sure, have fixed

**Comment by sergeyklay** [comment]
Thank you

**Pull request merged**